exporters/zipkin/src/recordable.cc

```diff
@@ -215,7 +224,17 @@ void Recordable::SetDuration(std::chrono::nanoseconds duration) noexcept
   span_["duration"] = duration.count();
 }
 
-void Recordable::SetSpanKind(opentelemetry::trace::SpanKind span_kind) noexcept {}
+void Recordable::SetSpanKind(opentelemetry::trace::SpanKind span_kind) noexcept
+{
+  if (kSpanKindMap.find(span_kind) != kSpanKindMap.end())
+  {
+    span_["kind"] = kSpanKindMap.at(span_kind);
```

Comment:
nit: save the `find()` result to a variable to avoid search the map twice?

Reply:
Good point. fixed that.